docs/docs/get-started/get-started.md

```diff
@@ -0,0 +1,82 @@
+---
+id: get-started
+title: Kurtosis Documentation
+slug: '/'
+sidebar_position: 1
+hide_table_of_contents: true
+---
+
+[Kurtosis](https://github.com/kurtosis-tech/kurtosis) is a tool for packaging and launching environments of containerized services where you want them, the way you want them, with one liners.
+
+- Get started with a [quickstart](quickstart.md) to launch an environment.
+- Dive deeper with [basic Kurtosis concepts](basic-concepts.md).
+- Write your own environment definition with a guide on [writing a package](write-your-first-package.md).
+
+To quickly see what Kurtosis feels like, check out the example snippets below:
```

Comment:
I wonder if we should have a very simple 

```py
def run(plan, args):
     plan.add_service(...a visual service)
```

Just to show how its dead simple to write one. Borrowing from Streamlit

Reply:
putting this on next iteration! i do want to take a position of extreme brevity on this page in particular, so i would opt to have the "writing a package" guide start with something very simple here. minimizing amount of info as much as possible